### PR TITLE
[risk=low][no ticket] move admin-user pages to admin/user

### PR DESCRIPTION
**Pull request by jmthibault79**
Rename and move some UI files related to User Admin pages
- rename admin-user-access to batch-sync-user-access
- rename admin-users to admin-user-table
- rename admin-user to deprecated-admin-user

Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: ui/src/app/routing/signed-in-app-routing.tsx [hunk 1]
```diff
@@ -50,10 +50,6 @@ const AccessRenewalPage = fp.flow(
   withRoutingSpinner
 )(AccessRenewal);
 const AdminBannerPage = fp.flow(withRouteData, withRoutingSpinner)(AdminBanner);
-const AdminNotebookViewPage = fp.flow(
```
Comment (jmthibault79) [inline ui/src/app/routing/signed-in-app-routing.tsx:53]: moved down to alphabetize

File: ui/src/app/pages/admin/batch-sync-user-access.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ import { SpinnerOverlay } from 'app/components/spinners';
 import { WithSpinnerOverlayProps } from 'app/components/with-spinner-overlay';
 import { userAdminApi } from 'app/services/swagger-fetch-clients';
 import { reactStyles } from 'app/utils';
-export const AdminUserAccess = (spinnerProps: WithSpinnerOverlayProps) => {
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/batch-sync-user-access.tsx:14]: renamed so as not to confuse with the all-users AdminTable page and the individual-user AdminUserProfile page.  This represents an admin action to perform on users.

File: ui/src/app/pages/admin/user/deprecated-admin-user.tsx [hunk 3]
```diff
@@ -130,7 +130,7 @@ interface State {
   verifiedInstitutionOptions: Array<PublicInstitutionDetails>;
 }
 
-export const AdminUser = withRouter(
+export const DeprecatedAdminUser = withRouter(
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/user/deprecated-admin-user.tsx:133]: to be removed soon: https://precisionmedicineinitiative.atlassian.net/browse/RW-8235

**Comment by jmthibault79** [comment]
e2e errors are mainly cohorts.  We should retest after this thread is resolved: https://pmi-engteam.slack.com/archives/C7JUTPQLE/p1650637761971419?thread_ts=1650637662.764409&cid=C7JUTPQLE

**Review by dmohs — APPROVED** [review]
Very nice.

**Pull request merged**